pkg/storage/stores/shipper/compactor/table.go

```diff
@@ -58,7 +58,8 @@ import (
 ////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 
 const (
-	uploaderName = "compactor"
+	uploaderName               = "compactor"
+	uploadIndexSetsConcurrency = 10
```

Comment:
Do you see any value in making this configurable? I wonder if folks with large numbers of tenants (i.e. us) would benefit from tuning this value.

Reply:
Works for me :+1: thanks for your thoughts.